Update included Sprig template functions (#8592)
open-cluster-management/rhacm-docs: governance/template_functions.adoc

[#open-source-community-functions]
=== Sprig open source

> {acm-short} supports all template functions from the `sprig` open source project, except for `env` and `expandenv`. This includes the following template functions:

Reviewer: Made a small update because there are functions that are not supported
```suggestion
{acm-short} supports most of the template functions from the `sprig` open source project, except for `env` and `expandenv`. View the following table:
```
